netweaver/monitoring.sls

```diff
@@ -20,17 +23,37 @@ sap_host_exporter_configuration_{{ exporter_instance }}:
          sap-control-uds: /tmp/.sapstream5{{ sap_instance_nr }}13
     - require:
       - pkg: prometheus_sap_host_exporter_pkg
-      - netweaver_install_{{ instance_name }}
+# on HA use case deploy ASCS|ERS on ERS|ASCS
+{% if netweaver.ha_enabled and node.sap_instance in ["ascs", "ers"] %}
+    - onlyif:
+      - test -d /usr/sap/{{ node.sid }}/ASCS* || test -d /usr/sap/{{ node.sid }}/ERS*
+{% endif %}
+
+# do not enable ASCS and ERS exporter service in HA use case (handled by pacemaker)
+{% if netweaver.ha_enabled and node.sap_instance in ['ascs', 'ers'] %}
```

Comment:
If we store the output of this condition, maybe we could put these values as one lines in the state itself

Reply:
@arbulu89 refactored this in 15eecaf